README.md

```diff
@@ -46,10 +46,18 @@ client.excluded?(duns: '080037478')
 ### Verify Vendor is a small business
 
 ```ruby
-client.small_business?(duns: '080037478', naicsCode: 541511)
+client.small_business?(duns: '080037478')
 #=> false
 ```
 
+This method checks against the following NAICS codes:
```

Comment:
what is the significance of these? (I am not sure what NAICS means) -- perhaps expand upon in a little here?